cmd/test_load.go

```diff
@@ -100,12 +104,25 @@ func (lt *loadedTest) initializeFirstRunner(gs *globalState) error {
 		testType = detectTestType(lt.source.Data)
 	}
 
+	state := &lib.RuntimeState{
+		Logger:         gs.logger,
+		RuntimeOptions: lt.runtimeOptions,
+		BuiltinMetrics: lt.builtInMetrics,
+		Registry:       lt.metricsRegistry,
+	}
+	if lt.runtimeOptions.KeyWriter.Valid {
```

Comment:
```suggestion
	if lt.runtimeOptions.KeyWriter.Valid {
		logger.Debugf("SSLKEYLOGFILE was specified, logging TLS connection keys to '%s'...", lt.runtimeOptions.KeyWriter.String)
```

I wonder if this shouldn't be a warning message even :thinking:

Reply:
Hmm maybe - this is from some of the previous PRs - haven't really looked at most of this messages :thinking:

Reply:
And yeah - I will make it a warning